js/map.js

```diff
@@ -112,7 +112,8 @@ function addMarker(map, nodeResult, name, location) {
 	
 	//If direction is given set it to the correct direction
 	if (nodeResult['cardinalDirection'] != null) ArrowDirection = nodeResult['cardinalDirection'];
-
+    if (nodeResult['cardinalDirectionAntenna'] != null) ArrowDirection = nodeResult['cardinalDirectionAntenna'];
```

Comment:
- Can this be written more concisely where the var has a condition, or function, that returns that appropriate direction property? Reduce the repetition of `ArrowDirection`
- Missing indent for alignment with `line 114`

Reply:
ArrowDirection is the variable that is used to decide what image will be displayed.

Defaults to omni on line 111

line 114 and 115 are both OPTIONAL parameters, if they are set it re-writes the ArrowDirection  accordingly

Could re-write it as a conditional if statement but that would take up more lines :P


Thoughts?
```
	if (nodeResult['cardinalDirectionAntenna'] != null) {
		ArrowDirection = nodeResult['cardinalDirectionAntenna'];
	} else if (nodeResult['cardinalDirection'] != null) {
		ArrowDirection = nodeResult['cardinalDirection'];
	}
```

Reply:
It might be more clear to have that conditional logic, and default params, wrapped in a single function. It's a stylistic thing that we can revisit. Thinking ahead to refactoring and linting.

I had something like this in mind, called in a variable:
```
function ArrowDirectionValue (nodeResult) {
  if (nodeResult['cardinalDirectionAntenna']) {
    return nodeResult['cardinalDirectionAntenna'];
  } else if (nodeResult['cardinalDirection']) {
    return nodeResult['cardinalDirection'];
  } else {
    return 'omni';
  }
}
```

Reply:
The code is used to draw a point in the ADD NODE function, its specific to that function so its used only once.  Making a function seems kinda of odd.  I cant think of any other reason why one would want to return a antenna direction over the node direction except to draw it.

But i defer to you expertise. I'm game for a function if you think its best.